packages/wrangler/src/api/startDevWorker/ConfigController.ts

```diff
@@ -204,6 +204,10 @@ async function resolveConfig(
 			legacyAssets: Boolean(legacyAssets),
 			script: input.entrypoint,
 			moduleRoot: input.build?.moduleRoot,
+			// getEntry only needs to know if experimental_assets was specified.
```

Comment:
If that's true, might I suggest flattening this to just a boolean?

Reply:
that's very fair! fixed ✅